test/core/util/test_config.cc

```diff
@@ -90,11 +92,32 @@ gpr_timespec grpc_timeout_milliseconds_to_deadline(int64_t time_ms) {
           GPR_TIMESPAN));
 }
 
-void grpc_test_init(int /*argc*/, char** argv) {
+namespace {
+void grpc_test_init_parse_argv(int* argc, char** argv) {
+  // flags to look for and consume
+  constexpr char poller_flag[] = "--poller=";
```

Comment:
Suggest using `absl::string_view` instead of `char[]` here.  That way, there's no need for a separate `size_t`, and the comparison in the loop becomes more readable using `absl::StartsWith()`.

Reply:
Done. To keep this as lightweight as possible given this code's use in most tests, I decided against using absl's ConsumePrefix or StripPrefix since we'd need to convert string_views to strings, then c_strs.